packages/itwin/geo-tools/package.json

```diff
@@ -0,0 +1,181 @@
+{
+  "name": "@itwin/geo-tools-react",
+  "version": "1.0.0",
+  "description": "React Geospatial Tools",
+  "keywords": [
+    "Bentley",
+    "iTwin.js",
+    "Geo",
+    "Bing",
+    "Map",
+    "Search",
+    "Address"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/bentley/geo-tools"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/geo-tools.js",
+  "module": "lib/esm/map-layers.js",
+  "typings": "lib/cjs/geo-tools",
+  "scripts": {
+    "build": "npm run -s copy:locale && npm run -s build:cjs",
+    "build:ci": "npm run -s build && npm run -s build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs && npm run -s copy:cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm && npm run -s copy:esm",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "copy:locale": "cpx -v \"./src/public/**/*\" ./lib/public",
+    "copy:cjs": "cpx -v \"./src/**/*.{*css,json,svg}\" \"./lib/cjs\"",
+    "copy:esm": "cpx \"./src/**/*.{*css,json,svg}\" \"./lib/esm\"",
+    "cover": "nyc npm -s test",
+    "docs": "",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "pseudolocalize": "betools pseudolocalize --englishDir=./src/public/locales/en --out=./src/public/locales/en-PSEUDO",
+    "test": "mocha \"./lib/cjs/test/**/*.test.js\""
+  },
+  "dependencies": {
+    "classnames": "^2.3.1",
+    "i18next": "^10.2.2",
+    "react-autosuggest": "^10.0.2",
+    "svg-sprite-loader": "4.2.1"
```

Comment:
lets try not to use the svg?sprite in this pkg, forces us to use webpack, and will make it much much harder to be an extension.

this should be one of the earlier pkgs to convert to extensions

Reply:
@FlyersPh9 could you add [this icon](https://github.com/iTwin/viewer-components-react/blob/fa390b3941841f3aec8724284396e644cb8e434a/packages/itwin/geo-tools/src/icons/geosearch.svg) to itwinui-icons?

Reply:
Thanks @FlyersPh9.

@mdastous-bentley The icon has been added, [see](https://itwin.github.io/[iTwinUI-icons](https://itwin.github.io/iTwinUI-icons/#geosearch)/#geosearch). You should be able to use the latest itwinui-icons-react pkg, youll want to use SvgGeosearch

Reply:
will remove svg?srpite usage in separate pr